tests/mesh/mesh_input.C

```diff
@@ -45,6 +45,54 @@ Number sin_x_plus_cos_y (const Point& p,
   return sin(x) + cos(y);
 }
 
+constexpr unsigned int denominator_for_side_elems = 12;
+
+Number designed_for_side_elems (const Point & p,
+                                const Parameters & param,
+                                const std::string &,
+                                const std::string &)
+{
+  const Real & x = p(0);
+  const Real & y = p(1);
+  const Real & z = p(2);
+
+  short facedim = param.have_parameter<short>("face") ?
+    param.get<short>("face") : -1;
+
+  // What face are we on?
+  auto is_on_face = [facedim](Real r, short rdim) {
+    if (facedim == rdim)
+      return true;
+    if (facedim >= 0)
+      return false;
+    const Real numerator = r * denominator_for_side_elems;
+    return (std::abs(numerator - std::round(numerator)) <
+            TOLERANCE*TOLERANCE);
+  };
+
+
+
+  // x/y/z components of a div-free flux,
+  // curl([x^2yz, xy^2z, xyz])
+  if (is_on_face(x, 0))
+    {
+      libmesh_assert(!is_on_face(y, 1));
+      libmesh_assert(!is_on_face(z, 2));
+//      return (x*z-x*y*y);
+return 3000+100*x+10*y+z;
```

Comment:
Does this line have weird indentation for any particular reason? Also maybe drop the commented out line if it isn't needed or was part of some debugging work, etc.

Reply:
Just the opposite - I use completely-unindented lines for debugging work, on the theory that they stick out so badly I won't fail to notice them when I'm skimming over `git diff --staged` before a commit.  This theory is not always correct, clearly.